files/templates/additional-scrape-configs.template.yaml

```diff
@@ -28,7 +28,11 @@
     replacement: {{ .APIServerURL }}:443
   - source_labels: [__meta_kubernetes_node_name]
     target_label: __metrics_path__
+{{ if eq .ClusterType "control_plane" }}
+    replacement: /api/v1/nodes/${1}:9393/proxy/metrics
```

Comment:
why 9393 when the port is 9323? https://github.com/giantswarm/giantnetes-terraform/blob/5982dee3a754a43c5ca25749c86f614a90f18a62/templates/vault.yaml.tmpl#L59

Reply:
You changed it already, but the port on the tenant clusters should be 9323 and should only work for clusters running 12.0.0+.
Docker is advising to use 9323 https://docs.docker.com/config/daemon/prometheus/#configure-docker so I updated that value in k8scloudconfig